test/integration/src/main/java/io/pravega/test/integration/selftest/adapters/InProcessMockClientAdapter.java

```diff
@@ -79,6 +79,7 @@
     //region Members
 
     private static final String LISTENING_ADDRESS = "localhost";
+    private static final String TLS_PROTOCOL_VERSION = "TLSv1.2,TLSv1.3";
```

Comment:
Why is this repeated in every single test? Don't we have a `TestUtils` common class which we can reference here?

Reply:
Removed this and referred to  `SecurityConfigDefaults` common class in all tests. Thanks.